package.json

```diff
@@ -25,7 +25,8 @@
   "module": "./dist/css-vendor.esm.js",
   "files": [
     "dist",
-    "src"
+    "src",
+    "LICENSE"
```

Comment:
License will always be published: https://docs.npmjs.com/files/package.json.html#files

Reply:
cool, didn't know